notebooks/embedding/20240327-is-dino.ipynb

```diff
@@ -0,0 +1,173 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    df_transformed = df.limit(24).withColumn("transformed_image", apply_dino_pbudf(df["data"]))
</pre><p>It probably makes sense to cache <code>df.limit(24)</code> into a separate variable e.g. <code>df_subset</code> so you dont have to go to disk everytime you want to rerun this.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dsgt-kaggle-clef/plantclef-2024/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>